controllers/suite_test.go

```diff
@@ -52,12 +52,12 @@ func TestMain(m *testing.M) {
 	env = envtest.New()
 
 	// Set up the MachineNodeIndex
-	if err := noderefutil.AddMachineNodeIndex(ctx, env.Manager); err != nil {
+	if err := index.AddMachineNodeIndex(ctx, env.Manager); err != nil {
 		panic(fmt.Sprintf("unable to setup machine node index: %v", err))
 	}
 
 	// Set up the MachineProviderIDIndex
-	if err := noderefutil.AddMachineProviderIDIndex(ctx, env.Manager); err != nil {
+	if err := index.AddMachineProviderIDIndex(ctx, env.Manager); err != nil {
 		panic(fmt.Sprintf("unable to setup machine providerID index: %v", err))
 	}
```

Comment:
What about creating AddDefaultIndexes in the new package? This can be re-used in main.go as well in few other test suites...